src/experimental/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,296 @@
+import React, {Children, useEffect, useRef, useState} from 'react'
+import Box from '../../Box'
+import sx, {SxProp} from '../../sx'
+import {useId} from '../../hooks/useId'
+import {invariant} from '../../utils/invariant'
+import {warning} from '../../utils/warning'
+import styled from 'styled-components'
+import {get} from '../../constants'
+import {ComponentProps} from '../../utils/types'
+import {getAnchoredPosition} from '@primer/behaviors'
+import type {AnchorSide, AnchorAlignment} from '@primer/behaviors'
+import '@oddbird/popover-polyfill'
+
+const StyledTooltip = styled.div`
+  // tooltip element itself
+  position: relative;
+  padding: 0.5em 0.75em;
+  font: normal normal 11px/1.5 ${get('fonts.normal')};
+  -webkit-font-smoothing: subpixel-antialiased;
+  color: ${get('colors.fg.onEmphasis')};
+  text-align: center;
+  text-decoration: none;
+  text-shadow: none;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: break-word;
+  white-space: normal;
+  background: ${get('colors.neutral.emphasisPlus')}; //bg--emphasis-color
+  border-radius: ${get('radii.2')};
+  border: 0;
+  width: max-content;
+  opacity: 0;
+  max-width: 250px;
```

Comment:
```suggestion
  max-width: 192px;
```

I'm adding some CSS vars for this, will let you know when it ships 😄